storybook/pages/ProfileShowcaseDirtyStatePage.qml

```diff
@@ -0,0 +1,197 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import StatusQ 0.1
+import StatusQ.Core.Utils 0.1
+
+import Storybook 1.0
+
+import AppLayouts.Profile.helpers 1.0
+
+Item {
+    id: root
+
+    ListModel {
+        id: communitiesModel
+
+        ListElement { key: "1"; name: "Crypto Kitties" }
+        ListElement { key: "2"; name: "Status" }
+        ListElement { key: "3"; name: "Fun Stuff" }
+        ListElement { key: "4"; name: "Other Stuff" }
+    }
+
+    ListModel {
+        id: communitiesShowcaseModel
+
+        ListElement { key: "1"; visibility: 1; position: 0 }
```

Comment:
Is it possible to use `ShowcaseVisibility` enum for visibility here?

Reply:
done